platform/integrations/slack_integration/setup.mdx

```diff
@@ -0,0 +1,163 @@
+For private channels, you need to invite the Tolgee app manually by using the command `/invite @Tolgee`.
+
+
+Now you need to link your Tolgee account with the Slack workspace:
+1. Type `/tolgee login` in the chat room to which the Tolgee app is invited.
+2. Button **Connect Tolgee account** will appear in the chat.
+
+<ScreenshotWrapper src="/img/docs/platform/slack_integration/slack_login.png" />
+
+3. Click on it and you will be redirected to the Tolgee login page.
+
+<ScreenshotWrapper src="/img/docs/platform/slack_integration/tolgee_login.png" />
+
+4. Then click **Link Accounts**.
+
+After login you will see a success message in the chat.
+
+Now you can use the Tolgee app in your Slack workspace.
+
+
+## Setup For Self-Hosted Users
+
+If you are using the self-hosted version of Tolgee, you also can use the Slack integration. The setup is a bit different than for cloud users.
+
+### Step 1: Create and Install the Tolgee App in Slack
+
+First of all you need to create a new app in your Slack workspace. You can do that by navigating to [Slack API](https://api.slack.com/apps)
+and clicking on **Create New App**. To know more about creating an app in Slack, you can refer to the [official documentation](https://api.slack.com/quickstart).
+After creation, you need to install the app in your workspace:
+
+1. Navigate to your app **Basic information** page
+2. Click on **Install to Workspace** button
+
+<ScreenshotWrapper src="/img/docs/platform/slack_integration/install_self_host.png" />
+
+
+### Step 2: Configure the App
+
+Now you need to provide all necessary tokens. First of all provide `signing-secret`.
+
+1. Go to the **Basic information** page of your app and find the **Signing Secret**.
+2. Navigate to your Tolgee instance and open `application.yaml` file.
+3. Add the following configuration:
+```yaml
+tolgee:
+ slack:
+    signing-secret: <signing-secret>
+```
+
+Your next step depends on whether you are using the Slack with single workspace or with multiple workspaces.
+In most cases self-hosted instances don't need to connect to multiple slack workspaces.
+
+#### Single Workspace
+
+1. First you need to provide all necessary bot Scopes. Navigate to **OAuth & Permissions** page and add the following scopes:
+   - `channels:history`
+   - `channels:read`
+   - `chat:write`
+   - `chat:write.customize`
+   - `commands`
+   - `groups:history`
+   - `groups:read`
+   - `im:history`
+   - `im:read`
+   - `mpim:history`
+   - `mpim:read`
+   - `users:read`
+
+2. Then reinstall the app in your workspace by clicking on **Reinstall App** button. This is necessary to apply the new scopes.
+Now you can copy the **Bot User OAuth Token**. It starts with `xoxb-`.
+
+3. Now you need to provide the token to your Tolgee instance. Navigate to your Tolgee instance and open `application.yaml` file.
+Add the following configuration:
+```yaml
+tolgee:
+ slack:
+    bot-token: <your-bot-token>
```

Comment:
As stated below, we can remove the multi workspace setup. So we can merge the configuration together, so user can configure it in one step.